src/asset-management/AaveManager.sol

```diff
@@ -214,7 +214,7 @@ contract AaveManager is IAssetManager, Owned(msg.sender), ReentrancyGuard {
         IAssetManagedPair lPair = IAssetManagedPair(msg.sender);
         int256 lAmount0Change = -int256(aToken0 ? aAmount : 0);
         int256 lAmount1Change = -int256(aToken0 ? 0 : aAmount);
-        assert(lAmount0Change < 0 || lAmount1Change < 0);
+        require(lAmount0Change < 0 || lAmount1Change < 0, "AM: RETURN_ASSET_BOTH_TOKENS");
```

Comment:
Is this a require or just sanity checking during dev? Like how can this ever trigger?

Reply:
with the current ternary logic we have right above, this should never trigger.

I think that this is a good sanity check to have since the pair only requests for asset returns one at a time. So this makes sure that the adjustManagement is only called for one asset

Reply:
But its quite literally impossible to trigger right? There exists no way/call to trigger this require?

require(amount > 0);
lAmountChange = -amount;
require(lAmountChange < 0);

should be true for all inputs?

Reply:
yup, I think with the symbolic execution power of our brains we can conclude that this would indeed never trigger. Removed the `require` statement

Reply:
Haha, I tried pyrometer but couldnt get it to work on our contracts